src/pages/Tab/Applications.vue

```diff
@@ -195,27 +195,48 @@
             </div>
           </div>
 
-          <div class="col-12 q-gutter-y-md">
-            <div v-for="(question, index) in form.questions" :key="index">
-              <div class="text-weight-bold text-primary q-my-sm">
-                Question {{ index + 1 }}
-                <q-btn
-                  dense
-                  flat
-                  label="Delete"
-                  text-color="red"
-                  @click="onClickDeleteQuestion(index)"
-                />
+      <q-separator class="q-my-md" />
+
+            <div class="col-12 q-gutter-y-md">
+              <div
+                v-for="(question, index) in form.questions"
+                :key="index"
+                class="q-pa-md bg-light-blue-1"
+              >
+                <div
+                  class="row text-weight-bold text-primary q-my-sm justify-between items-center"
+                >
+                  Question {{ index + 1 }}
+                  <q-btn
+                    dense
+                    flat
+                    label="Delete"
+                    text-color="red"
+                    @click="onClickDeleteQuestion(index)"
+                  />
+                </div>
+                <q-editor filled v-model="form.questions[index].text" />
               </div>
-              <q-editor filled v-model="form.questions[index].text" />
             </div>
-          </div>
+
+            <div class="col-12 q-pt-lg">
```

Comment:
the spacing is too big, use `q-pt-md` instead

![image](https://user-images.githubusercontent.com/22128538/106498546-6ee47680-64fa-11eb-86e4-30d6020cae46.png)